BlazarUI/app/scripts/components/sidebar/BuildsSidebarContainer.jsx

```diff
@@ -56,17 +57,30 @@ class BuildsSidebarContainer extends Component {
   }
 
   render() {
+    let miniLogo;
+    if (this.props.isCollapsed) {
+      miniLogo = <SidebarLogo mini={true} />
+    }
     // BuildsNotifier.updateModules(this.state.builds.modules);
     return (
-      <Sidebar>
-        <BuildsSidebar
-          builds={this.state.builds}
-          stars={this.state.stars}
-          loading={this.state.loading} 
-          persistStarChange={this.persistStarChange} />
-      </Sidebar>
+      <div>
+        {miniLogo}
```

Comment:
Rather than putting conditional logic in `render`, use a helper like `this.renderMiniLogo()`

```javascript
renderMiniLogo() {
  if (!this.props.isCollpased) {
    return null;
  }

  return (
    <SidebarLogo mini={true} />
  );
}
```